data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -450,6 +464,20 @@ spec:
                           type: string
                         type: array
                     type: object
+                  kubevirt:
+                    description: Kubevirt is the configuration used when installing on Kubevirt.
+                    properties:
+                      cpu:
+                        description: CPU is the mount of cpus used
```

Comment:
mount/amount - need to change in go code and regenerate

Reply:
Fixed